### DISCUSSION TITLE
HDR content doesn't render the same as on Windows

### DISCUSSION THREAD

**Discussion by nighv0120** [ChefKissInc/NootRX#75] [General]
### macOS Version

Sonoma

### What is your CPU's model?

AMD Ryzen 5 5600x

### What is your GPU's model?

AMD Radeon RX 6700XT

### Please describe the feature in detail.

The same monitor has a different effect on watching HDR videos, and I hope to add support for HDR

Mac OS
![IMG_20240815_125329](https://github.com/user-attachments/assets/3b851bfc-ac6d-41aa-a79b-eaba3f3a735a)
Windows10
![IMG_20240815_125120](https://github.com/user-attachments/assets/fbf4e56d-296f-4d56-adba-4019ec6ef839)

### What alternatives have you tried, if any?

I've tried turning on both the monitor's hdr and MacOS's high dynamic range at the same time

**Comment by VisualEhrmanntraut** [comment]
This is likely related to #4 or is user error.

**Comment by VisualEhrmanntraut** [comment]
Note that browsers other than Safari are unlikely to properly support HDR content on macOS, and also this isn't a real driver and is only adding in the missing pieces to existing drivers, so if the same thing happens to another card that's Navi 21 or Navi 23 then it's a problem with macOS and not NootRX.

**Comment by VisualEhrmanntraut** [comment]
Also, it would be helpful if you would provide a link to that test content so that the issue can be further diagnosed.

**Comment by nighv0120** [comment]
> 请注意，Safari 以外的浏览器不太可能在 macOS 上正确支持 HDR 内容，而且这不是真正的驱动程序，只是在现有驱动程序中添加缺失的部分，因此，如果另一张 Navi 21 或 Navi 23 卡也发生同样的事情，那么这就是 macOS 的问题，而不是 NootRX 的问题。

I use MacOS to watch this hdr video with Safari and Chrome browsers with the same effect, I can't see the middle word, and I can see the middle word when I watch this hdr video with Chrome and Edge browsers on Windows 10

**Comment by nighv0120** [comment]
> 此外，如果您能提供该测试内容的链接，以便进一步诊断问题，这将会很有帮助。

https://www.bilibili.com/video/BV1ta4y1L7jY/?spm_id_from=333.337.search-card.all.click&vd_source=ec520f2e3631f0ac69712a45ad5ff7fd

**Comment by nighv0120** [comment]
> 请注意，Safari 以外的浏览器不太可能在 macOS 上正确支持 HDR 内容，而且这不是真正的驱动程序，只是在现有驱动程序中添加补充的部分，因此，如果另一张 Navi 21 或 Navi 23也发生同样的事情，那么这就是 macOS 的问题，而不是 NootRX 的问题。

Thanks to you for making the driver to make my graphics card work on MacOS, I just found this problem and would like to ask you

**Comment by VisualEhrmanntraut** [comment]
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/1a528c9c-f40e-48e4-86ba-f6fc9982e81a">

This is how it renders on my M3 MacBook Pro. It seems to just be a macOS thing?

**Comment by nighv0120** [comment]
> <img alt="图像" width="1512" src="https://private-user-images.githubusercontent.com/30368284/358592619-1a528c9c-f40e-48e4-86ba-f6fc9982e81a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM4MTUwNjIsIm5iZiI6MTcyMzgxNDc2MiwicGF0aCI6Ii8zMDM2ODI4NC8zNTg1OTI2MTktMWE1MjhjOWMtZjQwZS00OGU0LTg2YmEtZjZmYzk5ODJlODFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE2VDEzMjYwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZlODFjNzlhMjc5ZmE2ZmJlZjY1M2NlZTRjNTM1NzgzOTk1ZWZiNTg1OWM1YTcwZGQ1ZGViOGYzNzU0ZmIxYjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.OA-3sEjysZMH2ZG0MIS_ZSAdIrS6LmViHSe8QSyf7Io">
> 这是在我的 M3 MacBook Pro 上的呈现效果。这似乎只是 macOS 的问题？

Thank you for your reply, I asked about the same 6700xt GPU and use your rootrx driver to watch this hdr video and he can see the middle word, I think it should be my system problem

**Comment by VisualEhrmanntraut** [comment]
Interesting

**Comment by nighv0120** [comment]
> 有趣的

I changed the HDR setting of the monitor from automatic to force HDR on to see the words, but the effect doesn't seem to be very good

**Comment by VisualEhrmanntraut** [comment]
Well it doesn't render as bright as Windows on my M3 MacBook Pro either, and I also had my friend with Navi 23 try it out and he says a similar result as my MacBook Pro.

**Comment by nighv0120** [comment]
> 嗯，它在我的 M3 MacBook Pro 上呈现的效果也不如 Windows 明亮，而且我也让我使用 Navi 23 的朋友试用了它，他说结果与我的 MacBook Pro 相似。

Thank you, I see

**Comment by VisualEhrmanntraut** [comment]
It's probably the OS and the display profile. I hope this conversation was helpful! Thanks for your co-operation.

**Comment by nighv0120** [comment]
> 可能是操作系统和显示配置文件的问题。希望本次对话对您有所帮助！感谢您的合作。

Thank you
